fe/fe-core/src/main/java/com/starrocks/sql/ast/CreateRoutineLoadStmt.java

```diff
@@ -548,6 +562,22 @@ private void checkKafkaProperties() throws AnalysisException {
         if (kafkaOffsetsString != null) {
             analyzeKafkaOffsetProperty(kafkaOffsetsString, kafkaPartitionOffsets);
         }
+        String confluentSchemaRegistryUrlString = dataSourceProperties.get(CONFLUENT_SCHEMA_REGISTRY_URL);
```

Comment:
1. Does the schema registry is case sensitive or case insensitive?
2. What version of avro does the schema registry support?

Reply:
1. schema registry is case sensitive
2. I didn't find the documentation, but we used the schema registry lib provided by confluent. The probability of version mismatch is relatively low.